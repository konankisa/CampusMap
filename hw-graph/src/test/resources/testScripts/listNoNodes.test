# Creates an empty graph
# and lists the nodes

# Create a graph
CreateGraph graph

# List the nodes in the graph
ListNodes graph