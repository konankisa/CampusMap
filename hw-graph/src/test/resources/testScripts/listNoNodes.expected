# Creates an empty graph
# and lists the nodes

# Create a graph
created graph graph

# List the nodes in the graph
graph contains: